base/pkg/entry.jl

```diff
@@ -148,11 +148,12 @@ function status(io::IO, pkg::AbstractString, ver::VersionNumber, fix::Bool)
 end
 
 function clone(url::AbstractString, pkg::AbstractString)
-    info("Cloning $pkg from $url")
+    normalized_url = Git.normalize_url(url)
```

Comment:
Does this also rewrite urls when users explicitly use ssh, for example? If the user explicitly provides a url, it shouldn't be rewritten.